app/services/finance/ecf/assurance_report/presenter.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Finance
+  module ECF
+    class AssuranceReport
+      class Presenter
+        def initialize(statement_id, query: nil)
+          self.statement_id = statement_id
+          self.query        = query
+        end
+
+        def to_csv
+          participant_declarations
+        end
+
+        def participant_declarations
+          @participant_declarations ||= query.participant_declarations.map do |participant_declaration|
```

Comment:
I think there might be performance implications if we were to map through a large number of participant declarations and hold in memory, maybe we should invoke things lazily?